custom_components/myskoda/climate.py

```diff
@@ -205,19 +210,24 @@ def __init__(self, coordinator: MySkodaDataUpdateCoordinator, vin: str) -> None:
     def _air_conditioning(self) -> AirConditioning | None:
         return self.vehicle.air_conditioning
 
+    def _auxiliary_heating(self) -> AuxiliaryHeating | None:
+        return self.vehicle.auxiliary_heating
+
     @property
     def _target_temperature(self) -> TargetTemperature | None:
         """Return target temp object for auxiliary heater."""
-        if self.has_any_capability(
-            [
-                CapabilityId.AUXILIARY_HEATING_TEMPERATURE_SETTING,
-                CapabilityId.AIR_CONDITIONING_HEATING_SOURCE_AUXILIARY,
-            ]
+        target_temperature = None
+        if self.has_all_capabilities(
+            [CapabilityId.AUXILIARY_HEATING_TEMPERATURE_SETTING]
+        ):
+            if ac := self._auxiliary_heating():
+                target_temperature = ac.target_temperature
+        elif self.has_any_capability(
+            [CapabilityId.AIR_CONDITIONING_HEATING_SOURCE_AUXILIARY]
         ):
             if ac := self._air_conditioning():
                 target_temperature = ac.target_temperature
-                if target_temperature is not None:
-                    return target_temperature
+        return target_temperature
```

Comment:
This sensor only `is_supported` when any of these capabilities are present:
```python
[
    CapabilityId.AUXILIARY_HEATING,
    CapabilityId.AIR_CONDITIONING_HEATING_SOURCE_AUXILIARY,
]
```
Why do we care about the `None` case?